packages/dates/src/FormatDate/FormatDate.js

```diff
@@ -0,0 +1,59 @@
+import React, { Component } from 'react'
+import PropTypes from 'prop-types'
+import moment from 'moment'
+
+import { TextBase } from '@hixme-ui/text'
+
+const Time = TextBase.withComponent('time')
+
+class FormatDate extends Component {
+  static format(props) {
+    const {
+      date,
+      datetime,
+      format,
+      inputFormat,
+    } = props
+
+    let formatStr = format
+
+    if (!formatStr) {
+      formatStr = date ? 'MM/DD/YYYY' : 'MM/DD/YYYY h:mm:ss a'
+    }
+
+    return moment(date || datetime, inputFormat, true).format(formatStr)
+  }
+
+  static isValid(props) {
+    const { date, inputFormat } = props
+    if (!date || !inputFormat) { return false }
+
+    return moment(date, inputFormat, true).isValid()
+  }
+
+  render() {
+    const { ...rest } = this.props
```

Comment:
This should probably be 
`const { datetime, date, format, inputFormat, ...rest } = this.props` 

This way the DateFormat props aren't transferred in the spread for the TextBase

Reply:
Gotcha, thanks.